### PR TITLE
[Dubbo-1330] Fix Support MetaspaceSize and MaxMetaspaceSize vm args in java8+

### PR DESCRIPTION
**Pull request by web1992**
## What is the purpose of the change

see #1330 
- Support `MetaspaceSize` and `MaxMetaspaceSize ` in java8+

## Brief changelog

In linux and windows I use shell or batch to get java version by `java -fullversion`, you can see it in commit.

In the commit ,you can find I use `JAVA_8_VERSION="180"` to compare java vesion. The reasion is that java version naming rule change in java 9.

> `java -fullversion` output example  

- in jdk1.7 -> java full version "1.7.0_79-b15"
- in jdk1.7 -> java full version "1.8.0_152-b16"
- in jdk9 -> java full version "9.0.4+11"

I find many `dos` symbol in `start.sh`, if you run `start.sh` in linux, it may be a problem. I use `dos2unix` cmd to format the file.

## Verifying this change
Test result in windows 
- jdk9
image
- jdk1.8
image
- jdk1.7
image

### CODE REVIEW

File: dubbo-container/dubbo-container-api/src/main/resources/META-INF/assembly/bin/start.sh [hunk 1]
```diff
@@ -58,11 +60,18 @@ if [ "$1" = "jmx" ]; then
     JAVA_JMX_OPTS=" -Dcom.sun.management.jmxremote.port=1099 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false "
 fi
 JAVA_MEM_OPTS=""
+#set jvm args by different java version
+JAVA_VERSION=`java -version 2>&1 | awk -F[\"\.] -v OFS=. 'NR==1{print $3}'`
```
Comment (beiwei30) [inline dubbo-container/dubbo-container-api/src/main/resources/META-INF/assembly/bin/start.sh]: JAVA_VERSION=`java -version 2>&1 | grep -i 'java version' | awk -F[\"\.] -v OFS=. 'NR==1{print $3}'`

**Comment by beiwei30** [comment]
@web1992, I put some comments on your code change, pls. check.

**Comment by beiwei30** [comment]
@web1992 Your code for parsing java version is not robust, in fact it doesn't work in my environment :p. A better way to parse java version is `JAVA_VERSION=`java -version 2>&1 | grep -i 'java version' | awk -F[\"\.] -v OFS=. 'NR==1{print $3}'`

**Comment by web1992** [comment]
@beiwei30  What is you java version and os ? Mac ,Linux or Windws ?
I test in java 9 on linux ,the output is `0`
```shell
java -version 2>&1 | awk -F[\"\.] -v OFS=. 'NR==1{print $3}' 
# output is 0
```
image

**Comment by diecui1202** [comment]
@web1992 your shell is not work on my mac os. you can use this way:

```
    JAVA_VERSION=`$JAVA_HOME"/bin/java" -version 2>&1` || { echo $JAVA_HOME"/bin/java -version failed"; exit 1; }
    JAVA_VERSION_MAJOR=${JAVA_VERSION:16:1}
    if [ "$JAVA_VERSION_MAJOR" -ge 8 ]; then
        ......
```

It works on mac and linux.

**Pull request merged**